jsonnet/mixin/config.libsonnet

```
@@ -1,5 +1,7 @@
 {
   _config+:: {
     prometheusOperatorSelector: 'job="prometheus-operator"',
+    prometheusSelector: 'prometheus-k8s',
+    alertManagerSelector: 'alertmanager-main',
```

these are specific to the default kube-prometheus setup. I'd rather see a configReloaderSelector option defaulting to all namespaces?

```suggestion
    configReloaderSelector: 'namespace=~".+"',
```

---

makes sense, ty